docs/visual-basic/misc/codesnippet/VisualBasic/bc31080_1.vb

```diff
@@ -1 +0,0 @@
-        If Not A Is B Then
```

Comment:
This is really a nit unrelated to your PR, but in the source in the samples repo, it would be best to modernize this with:
`If A IsNot B Then`
I'll open a PR to do it.